### PR TITLE
update: dont unload if the user is in wonderdome

### PR DESCRIPTION
**Pull request by NeonGamerBot-QK**
um im terrible at nextjs so this might not work at first ;-;

### CODE REVIEW

**Comment by YeetTheAnson** [comment]
YUPPEE 
Hopefully this gets approved quickly

File: src/app/harbor/battles/battles.tsx [hunk 1]
```diff
@@ -319,7 +319,11 @@ export default function Matchups({ session }: { session: HsSession }) {
   const [isSubmitting, setIsSubmitting] = useState(false)
   const [cursed, setCursed] = useState(false)
   const [blessed, setBlessed] = useState(false)
-
+  useEffect(() => {
+    window.onbeforeunload = () => {
+      return Boolean(selectedProject)   
+ }
+})
```
Comment (malted) [inline src/app/harbor/battles/battles.tsx:326]:
```suggestion
}, [])
```

**Comment by maxwofford** [comment]
@NeonGamerBot-QK don't forget to `bun fmt!`

**Comment by NeonGamerBot-QK** [comment]
@maxwofford done!

**Comment by malted** [comment]
:D

**Comment by maxwofford** [comment]
@malted did you test this? https://github.com/hackclub/high-seas/issues/943

**Pull request merged**